examples/angular/src/server.ts

```diff
@@ -0,0 +1,39 @@
+///<reference types="node"/>
+import 'zone.js/dist/zone-node';
+
+import {ngExpressEngine} from '@nguniversal/express-engine';
+import * as express from 'express';
+import {join} from 'path';
+
+// Express server
+const app = express();
+
+const PORT = process.env.PORT || 4000;
+const DIST_FOLDER = join(process.cwd(), 'src/prodapp');
+
+import {AppServerModule} from './app/app.server.module';
+
+// Our Universal express-engine (found @
+// https://github.com/angular/universal/tree/master/modules/express-engine)
+app.engine('html', ngExpressEngine({
+             bootstrap: AppServerModule,
+             providers: [
+               // provideModuleMap(LAZY_MODULE_MAP)
```

Comment:
@Toxicable Do you have an idea how to generate the `LAZY_MODULE_MAP`?

Btw. I used this pull request as base and server side rendering works! (The only issue is that lazy loading requires the `LAZY_MODULE_MAP` and I don't know from to get it.